rpcd/playbooks/roles/beaver/templates/beaver.j2

```diff
@@ -60,6 +67,7 @@ do_stop() {
         I="$((I+1))"
     done
     echo '.'
+    test -f ${BEAVER_PID} && rm -f ${BEAVER_PID}
```

Comment:
Because beaver is not deleting the file after it stopped which is quite annoying.